website/orders/models.py

```diff
@@ -110,6 +112,15 @@ class Product(models.Model):
         max_digits=6, decimal_places=2, validators=[MinValueValidator(Decimal("0.00"))]
     )
 
+    orderable = models.BooleanField(
+        default=True, null=False, help_text="Whether or not this product should appear on the order page."
```

Comment:
Maybe refactor this to 'is_takeaway' or something, as in "a product that needs to be prepared".

Reply:
I thought of it as "can be ordered from the order page". Maybe rename to "can_be_ordered"?

We can namely also let the soda cans be ordered with the tosti's on the order page.

Reply:
But then they will also appear in the queue, right?

Reply:
Hmm, but it might be nice to order a soda can with your TOSTI on the webpage and also display the list of things that can be bought in the canteens. As of now it will appear on the order page. I suggest we change it to the following:

- We add a "show_on_shift_overview" BooleanField.
- Users are able to order soda cans on the order page.
- The orders of products with show_on_shift_overview False will result in the orders not being shown to all users, but rather only to the user that ordered the product.
- The admin view shows all orders: Tosti's, ordered soda and scanned soda.

Reply:
> "display the list of things that can be bought in the canteens" 
I agree with that!

Reply:
And what about this?

> * We add a "show_on_shift_overview" BooleanField.
> * Users are able to order soda cans on the order page.
> * The orders of products with show_on_shift_overview False will result in the orders not being shown to all users, but rather only to the user that ordered the product.
> * The admin view shows all orders: Tosti's, ordered soda and scanned soda.